var/spack/repos/builtin/packages/turbovnc/package.py

```diff
@@ -0,0 +1,116 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Turbovnc(CMakePackage):
+    """TurboVNC is a derivative of VNC (Virtual Network Computing) 
+ that is tuned to provide peak performance for 3D and video workloads.
+ TurboVNC was originally a fork of TightVNC 1.3.x, on the surface, 
+ the X server and Windows viewer still behave similarly to their parents."""
+    # FIXME: add a proper url for your package's homepage here.
+    homepage = "http://www.turbovnc.org/"
+    url      = "http://downloads.sourceforge.net/project/turbovnc/2.0.1/turbovnc-2.0.1.tar.gz"
+
+
+    version('2.1', '6748bb13647d318f0c932394f8298d10')
+    version('2.0.1', 'a279fdb9ac86a1ebe82f85ab68353dcc')
+
+
+    # FIXME: Add dependencies if this package requires them.
+    variant('java', default=False, description='Enable Java build')
+
+   
+    depends_on('cmake', type='build')
+    depends_on("libjpeg-turbo")
+    depends_on("libjpeg-turbo", when='+java')
```

Comment:
This doesn't make sense. You always depend on `libjpeg-turbo`, then on the next line you say to conditionally depend on `libjpeg-turbo`.